src/components/ChallengesPanel.js

```diff
@@ -77,22 +80,31 @@ const ChallengesPanel = ({
   headerType = 'h2'
 }) => {
   const Header = headerType;
+  const { isServer } = useSSR();
+  const [suggestions, setSuggestions] = useState(() =>
+    // Empty placeholders on server side render
+    challenges.map(() => ({})).slice(0, 2)
```

Comment:
I originally thought this was a little weird, but realized that you do this to make sure there's the same number of empty challenges as there are challenges. So I would make that clear in the comment, so someone doesn't replace this with a simple `[{}, {}]`

Reply:
Thanks a lot for the feedback, I added a more precise comment :)